completions/jump.completion.sh

```diff
@@ -0,0 +1,10 @@
+#! bash oh-my-bash.module
+
+if _omb_util_command_exists jump; then
+	_jump()
+	{
+	    local JUMP_COMMANDS=$(jump 2>&1 | command grep -E '^ +' | awk '{print $1}')
+	    COMPREPLY=( $(compgen -W "$JUMP_COMMANDS" -- "${COMP_WORDS[1]}") )
```

Comment:
```suggestion
	    COMPREPLY=( $(compgen -W '$JUMP_COMMANDS' -- "${COMP_WORDS[1]}") )
```

The option argument of `compgen -W` is subject to `eval`, so there is a problem with shell double expansions. `$...` should be singly quoted to avoid an extra expansion.